configure.cmd

```diff
@@ -85,12 +92,14 @@ REM Parse command line parameters
         ) else if /I "%1" == "Makefiles" (
             set CMAKE_GENERATOR="MinGW Makefiles"
         ) else if /I "%1" == "VSSolution" (
-            echo. && echo Error: Creation of VS Solution files is not supported in a MinGW environment.
+            echo. && echo Error: Creation of Visual Studio solution files is not supported in a MinGW environment.
             echo Please run this command in a [Developer] Command Prompt for Visual Studio.
+
             goto quit
         ) else if /I "%1" NEQ "" (
             echo.%1| find /I "-D" >nul 2>&1
-            if not errorlevel 1 (
+            
+			if not errorlevel 1 (
```

Comment:
You are changing a lot of whitespace around for no good reason?

Reply:
Tabs detected on this line.